modules/040-node-manager/hooks/handle_draining.go

```diff
@@ -70,15 +68,33 @@ func drainFilter(obj *unstructured.Unstructured) (go_hook.FilterResult, error) {
 		return nil, err
 	}
 
-	var isDraining bool
-	if _, ok := node.Annotations["update.node.deckhouse.io/draining"]; ok {
-		isDraining = true
+	var (
+		drainingSource string
+		drainedSource  string
+	)
+	if source, ok := node.Annotations["update.node.deckhouse.io/draining"]; ok {
```

Comment:
In hooks (not in tests) this annotation key is repeated 4 times. I suggest extracting it to a constant

Reply:
Done